src/tictactoe/game_runner.clj

```diff
@@ -0,0 +1,160 @@
+  [options]
+  (doseq [[number choice] 
+    (map identity options)] 
+      (print-message (name number) choice)))
+
+(defn- valid-choice? 
+  ([options choice]
+    (contains? options choice))
+  ([options other-marker choice]
+    (and (contains? options choice) (not= (string/lower-case other-marker) choice))))
+
+(defn- invalid-choice 
+  [errors options]
+  (print-message (:error errors))
+  (get-input))
+
+(defn- marker-errors
+  [options other-marker current-marker]
+  (if-not (valid-choice? options other-marker current-marker)
+    {:error "Invalid. Please pick another marker."}
+    {:error nil}))
+
+(defn- menu-errors
+  [options choice]
+  (if-not (valid-choice? options choice)
+    {:error "Invalid. Please pick an available option."}
+    {:error nil}))
+
+(defn- errors 
+  ([options choice]
+    (menu-errors options choice))
+  ([options other-marker choice]
+    (marker-errors options other-marker choice)))
+
+(defn- validate-markers
+  [current-marker available-markers other-marker]
+  (let [possible-error (errors available-markers other-marker current-marker)]
+    (if (:error possible-error)
+      (recur (invalid-choice possible-error available-markers) 
+        available-markers other-marker) current-marker)))
+
+(defn- validate-menu-choice
+  [choice options]
+  (let [possible-error (errors options choice)]
+    (if (:error possible-error)
+      (recur (invalid-choice possible-error options) options) choice)))
+
+(defn input-validation-loop 
+  "Takes a game option and message and validates the input recevieved from user is valid.
+    If input is invalid it will loop until valid input is received."
+    ([choice options]
+      (validate-menu-choice choice options))
+    ([choice options other-marker]
+      (validate-markers choice options other-marker)))
+
+(defn pick
+  "Takes a game option and message and prompts user for valid input and returns it."
+  [options options-message]
+  (print-message options-message)
+  (loop-and-print options)
+  (->
+    (get-input)
+    (input-validation-loop options)
+    (str)))
+
+(defn pick-marker 
+  "Prompts the user for player 1's maker and player 2's marker when argument is received"
+  ([]
+    (print-message (player-marker-message "player 1"))
+    (->
+      (get-input)
+      (string/lower-case)
+      (input-validation-loop markers)
+      (string/upper-case)))
+  ([player1-marker]
+    (print-message (player-marker-message "player 2"))
+    (->
+      (get-input)
+      (string/lower-case)
+      (input-validation-loop markers player1-marker)
+      (string/upper-case))))
+
+(defn chosen-game-options 
+  "Takes a hash-map of all game options chosen and displays the summary"
+  [game-options]
+  (print-message (game-choices-message))
+  (loop-and-print game-options))
+
+(defn menu
+  "Displays a welcome message and prompts the user for game type, board size, and player markers.
+    It displays the summary of options and returns a hash-map of choices."
```

Comment:
Strange indentation here. You'll want to indent your docstrings so when you call `(doc tictactoe.game-runner/menu)` at the repl it looks correctly formatted.

That's usually like this

```
(defn add
  "Returns the sum of two number. Oh look,
  a newline!"
  [x y]
  (+ x y))